src/utils/TedexSetup.ts

```diff
@@ -0,0 +1,59 @@
+import { TraderClient, TradeType } from 'tdex-sdk';
+import { toSatoshi } from './format';
+import BigNumber from 'bignumber.js';
+import { Market } from './constants';
+
+const client = new TraderClient('http://localhost:9945');
+
+export const handleMarkets = async () => {
+  try {
+    const markets = await client.markets();
+    console.log(markets);
+
+    return markets;
+  } catch (error) {
+    throw new Error('TDEX provider and markets is not reachable');
+  }
+};
+
+export const onSendAmountChange = async (
```

Comment:
title of function is not relevant. Its just a wrapped way to call `marketPrice`

Reply:
maybe just `Preview`

Reply:
fixed this